### ISSUE TITLE
Embedded markdown with latex equations and (embedded) images render incorrectly

### ISSUE THREAD

**Issue by jordigraumo** [zsviczian/obsidian-excalidraw-plugin#360]
**Describe the bug**
Embedded markdown notes don't show correctly when  containing equations ( e.g. $\sum_x p(x)$ ) or embedded images (e.g. ![[image.png]]).

**To Reproduce**
Create a an example note:

note.md
```
Some incorrectly rendered equation $\sum_x p(x)$
$$\sum_x p(x)$$

Some incorrectly rendered image 
![[image.jpeg]]
```

And use the command "Excalidraw: insert markdown file from vault" to insert the note.

What we see is that the equations are incorrectly rendered and the image is not rendered at all.

**Expected behavior**
The rendered note in a excalidraw canvas should contain the latex equation and image that looks like the obsidian preview of "note.md" above. 

**Screenshots**
Incorrect latex:
<img src="https://user-images.githubusercontent.com/16000893/149594863-c1a9f15e-fce8-4069-90b7-80fee48a8b9e.png" width=50% height=50%>

**Desktop (please complete the following information):**
 - OS: macOS 11.2
 - Browser Chrome

**Comment by zsviczian** [comment]
Yes, I am aware of that. Unfortunately, that is a limitation today. SVG images do work though, and even some code blocks like some of the dataview queries, but only a few. I might be able to solve the issue of embedded jpg and png images. LaTeX is a complex problem and will most likely not work ever as part of markdown embeds in this plugin. 

I am sorry, but at this time, I am not planning to work on this issue.
